cmd/collectors/rest/plugins/qtree/qtree.go

```diff
@@ -210,5 +206,9 @@ func (my *Qtree) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 		Int("metrics", numMetrics).
 		Msg("Collected")
 
-	return []*matrix.Matrix{my.data}, nil
+	// This would generate quota metrics prefix with `qtree_`. These are deprecated now and will be removed later.
+	qtreePluginData := collectors.CloneWithOtherIdentifier(my.Parent+".Qtree", "qtree", "qtree", my.data)
+	output = append(output, qtreePluginData)
```

Comment:
minor: these can be replaced with `	return []*matrix.Matrix{qtreePluginData, my.data}, nil`